### ISSUE TITLE
SQLAlchemy error

### ISSUE THREAD

**Issue by mike-bailey** [balidani/tinyctf-platform#12]
Getting:
```ProgrammingError: (ProgrammingError) SQLite objects created in a thread can only be used in that same thread.The object was created in thread id 139626447152896 and this is thread id 139626314942208 None None```

When trying to register. AWS EC2 image of 16.06.1 LTS. Followed steps, ran it with python server.py.

**Comment by geolado** [comment]
Same here

**Comment by mike-bailey** [comment]
FYI I changed the server to single thread and it worked as a band-aid

**Comment by geolado** [comment]
Yeah , that wouldn't work for me . Maybe if I run it over uwsgi is gonna work ?

**Comment by mike-bailey** [comment]
No idea
